cmd/nerdctl/push.go

```diff
@@ -61,6 +64,11 @@ func newPushCommand() *cobra.Command {
 	pushCommand.Flags().Bool("estargz", false, "Convert the image into eStargz")
 	pushCommand.Flags().Bool("ipfs-ensure-image", true, "Ensure the entire contents of the image is locally available before push")
 
+	pushCommand.Flags().Bool("sign", false, "Allow signing images right after built them")
```

Comment:
I prefer this to be --cosign , as we may  add other signing functionalities such as notary v2

Reply:
Also, please add docs (docs/cosign.md) and link it from the command reference in README.md

Reply:
And I prefer to mark this experimental (https://github.com/containerd/nerdctl/blob/master/docs/experimental.md)

Reply:
Done